src/homeblks/volume/volume.cpp

```diff
@@ -335,7 +335,105 @@ indx_tbl* Volume::recover_indx_tbl(btree_super_block& sb, btree_cp_sb& cp_info)
     return static_cast< indx_tbl* >(tbl);
 }
 
+static std::vector< std::pair< int, int > > get_true_intervals(const std::vector< bool >& empty_blocks) {
```

Comment:
I don't see any place using this api as well as copmute_range_intervals, can we remove them unless there is a plan to use them in the near future?